### ISSUE TITLE
ot-key directives aren't aware of element visibility

### ISSUE THREAD

**Issue by oTree-org** [qwiglydee/otree-front#47]
I have a task where each trial is submitted by pressing "enter" (or clicking the associated "submit" button):

    <button type="button" ot-input="submit = true" ot-key="Enter">submit</button>

I also have a feedback stage after each trial, where the user reads the correct answer and then proceeds to the next trial by pressing "enter" (or clicking the associated "next" button):

```
<main ot-if="is_feedback == true">
    [feedback shown here...]
    <button type="button" ot-input="next = true" ot-key="Enter">next trial</button>
</main>
```

Although these 2 buttons are never shown at the same time, it seems that both event handlers get triggered when I press enter. 

Is this the desired behavior? Would it be possible to unbind ot-key if the element is hidden?

**Comment by qwiglydee** [comment]
the workaround for now is to add 'ot-enabled="condition"' to the buttons, with the same conditions as of `ot-if`

**Comment by qwiglydee** [comment]
It's actually addressed by #44

Although the `ot-key` is not an input, it only works in combination of `ot-input/ot-event` which should provide disabled state.
